tests/codeception/functional/Install/Content/FlexiblePageCest.php

```diff
@@ -492,6 +492,10 @@ public function testCollections(FunctionalTester $I) {
     $I->selectOption('Items Per Row', 2);
     $I->canSeeOptionIsSelected('Background Color', '- None -');
     // Add a Text Area component to the Collection.
+    $I->selectOption('Title Settings', 'I do not want a heading for this Collection');
+    $I->cantSee('field_hs_page_components[1][subform][field_title][0][value]');
```

Comment:
You could use `cantSeeElement` too. I'm not sure how it checks to "see", ie is it just looking at the source of the page or is it looking at the rendered text.